kvmagent/kvmagent/plugins/qga_zwatch.py

```
@@ -116,16 +119,33 @@ def zwatch_qga_monitor(self):
+                nicInfoStatus = nicInfoStatus or qga.guest_file_is_exist(self.WIN_ZWATCH_GET_NIC_INFO_EXE_PATH)
             if not nicInfoStatus:
                 return
             if is_windows_2008(qga):
                 nicInfo = get_nic_info_for_windows_2008(uuid, qga)
+            elif qga.os and 'mswindows' in qga.os:
+                nicInfo = None
+                if qga.guest_file_is_exist(self.WIN_ZWATCH_GET_NIC_INFO_EXE_PATH):
+                    try:
+                        nicInfo = qga.guest_exec_program_no_exitcode(
+                            self.WIN_ZWATCH_GET_NIC_INFO_EXE_PATH, ['nic-info'])
+                    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)